hydra/core/override_parser/overrides_parser.py

```diff
@@ -28,6 +28,21 @@
     )
     sys.exit(1)
 
+# Mappings between lexer modes and parser rules.
+MODE_TO_RULES = {
+    "KEY": ["key", "override", "package", "packageOrGroup"],
+    "ARGS": [
```

Comment:
ARGS does not seem to correctly describe the mode.
Maybe KEY and NORMAL ?

Reply:
Right, this came from OmegaConf (where this mode is used for resolver arguments). What about VALUE? (NORMAL sounds a bit vague)

Reply:
sure, we can go with KEY and VALUE.

Reply:
Done in 19f11b09